app/models/movie.rb

```diff
@@ -1,2 +1,3 @@
 class Movie < ApplicationRecord
+  validates(:title, { :presence => true, :uniqueness => true })
```

Comment:
@raghubetina Is this the syntax you prefer validations should be rendered as? The account generator adds a couple in the modern syntax. I brought up the question [here](https://github.com/raghubetina/review_draft_generators_67/pull/1#discussion_r346938775) but forgot to follow up. I wasn't sure if you wanted to get them familiar with the new syntax this way.

Reply:
@jelaniwoods Yes, let's use the old verbose syntax. Sorry for the slow reply.